src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -379,12 +379,9 @@ private async Task<bool> PromptPackageIdentifierAndCheckDuplicates(Manifests man
         {
             GitHub client = new GitHub(this.GitHubToken, this.WingetRepoOwner, this.WingetRepo);
 
-            if (!string.IsNullOrEmpty(this.GitHubToken))
+            if (string.IsNullOrEmpty(this.GitHubToken))
             {
-                if (!await this.SetAndCheckGitHubToken())
-                {
-                    return false;
-                }
+                this.GitHubToken = await this.GetTokenFromCache();

```

Comment:
Doesn't make sense to do this after already creating the client?

Reply:
Removed, no longer needed.